src/components/Modal/Modal.vue

```diff
@@ -703,23 +703,29 @@ $header-size: 50px;
 	/* Content */
 	.modal-container {
 		display: block;
-		overflow: hidden;
+		overflow: auto; // avoids unecessary hacks if the content should be bigger than the modal
```

Comment:
Breaking change

Reply:
yes, I know, hence I've added the breaking label. But it is necessary to make this work reliably.

Reply:
If we're following SemVer then this would consitute a new major :)

Reply:
Jep. Would it be 4.1.0 or 5.0.0 ?

Reply:
5.0.0 :)

Reply:
Added milestone 5.0.0 :)